assets/styles.css

```diff
@@ -0,0 +1,408 @@
+  float: left;
+  width: 200px;
+}
+
+.footer-header a {
+  position: relative;
+  float: right;
+  text-align: right;
+  color: #567b95;
+  font-size: 12px;
+  width: 200px !important;
+}
+
+.editor-picks-article article img {
+  width: 80px;
+  margin-top: auto;
+}
+
+.editor-picks-article article {
+  display: flex;
+  margin: 10px auto;
+}
+
+.editor-picks-article article a {
+  padding: 10px;
+}
+
+.inflation {
+  width: 60vw;
+  margin: auto;
+  position: relative;
+}
+
+.inflation figure {
+  margin-top: 5px;
+}
+
+.thumbnail-texts {
+  position: absolute;
+  top: 14vw;
+}
+
+.thumbnail-texts section:nth-child(2) {
+  position: relative;
+  top: 6vw;
+}
+
+.thumbnail-texts p {
+  font-size: 0.7em;
+  width: 25%;
+  float: left;
+  padding: 8px;
+}
+
+.inflation hr {
+  width: 80px;
+}
+
+.inflation-footer {
+  font-size: 13px;
+}
+
+.youtube-section {
+  width: 40vw;
+  margin: 30px auto;
+}
+
+.youtube-section figure span {
+  color: #666;
+  font-size: 13px;
+}
+
+.spiral-image caption {
+  color: #666;
+  font-size: 10px;
+}
+
+.article-footer {
+  margin-top: 10px;
+}
+
+.article-footer small {
+  color: #666;
+}
+
+.more-section {
+  width: 90vw;
+  max-width: 1200px;
+  display: flex;
+  margin: 50px auto;
+}
+
+.grid-section h4,
+.popular h4 {
+  border-top: solid thin #333;
+  padding: 15px 0;
+}
+
+.grid-section {
+  flex-basis: 70%;
+}
+
+.grid-container {
+  display: -ms-grid;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 30px 15px;
+  margin-bottom: 30px;
+}
+
+.editors-picks {
+  margin-top: 30px;
+}
+
+.popular {
+  flex-basis: 30%;
+  padding-left: 20px;
+}
+
+.popular ul li {
+  list-style: none;
+  border-top: solid thin #e2e2e2;
+  padding: 15px 0;
+}
+
+.popular ul li a {
+  font-size: 19px;
+}
+
+.more-comments {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+.sub-footer {
+  display: flex;
+  width: 85vw;
+  max-width: 1200px;
+  margin: auto;
+  margin-bottom: 100px;
+}
+
+.sub-footer ul li a:hover {
+  text-decoration: underline;
+}
+
+.sub-footer-left {
+  display: flex;
+  width: 80%;
+  justify-content: space-between;
+  margin: 20px 0;
+}
+
+.sub-footer-left ul li {
+  list-style-type: none;
+  padding: 5px 0;
+  font-size: 14px;
+}
+
+.sub-footer-right {
+  width: 20%;
+  padding-left: 20px;
+  margin: 20px 40px;
+  border-left: solid thin #f5f5f5;
+}
+
+.sub-footer-right ul li {
+  list-style-type: none;
+  padding: 5px 0;
+}
+
+.border-bottom {
+  border-bottom: solid thin #f5f5f5;
+}
+
+.footer-header {
+  border-top-style: double;
+  border-color: #f1f1f1;
+  padding-top: 10px;
+  width: 85vw;
+  max-width: 1200px;
+  margin: auto;
+}
+
+.sticky-footer {
+  position: fixed;
+  background-color: #fff;
+  bottom: 0;
+  border-top: solid thin #333;
+  width: 100%;
+  padding: 10px 0;
+}
+
+.sticky-footer p {
+  text-align: center;
+  position: relative;
+  bottom: 10px;
+  font-size: 16px;
+}
+
+.sticky-footer button:hover {
+  cursor: pointer;
+}
```

Comment:
This selector has the same style definition of the selectors on lines 52 and 98. Group selectors with the same style definition or create a new class.